### DISCUSSION TITLE
Phishing website takedown request

### DISCUSSION THREAD

**Discussion by javierpignata** [vercel/community#6653] [Help] [answered]
### Summary

Hi team! The following URL is being used to perform a phishing attack:

[REDACTED]

It is a multi wallet phishing kit stealing seed phrases from people's crypto wallets:
![image](https://github.com/vercel/community/assets/96588126/58ffc0ca-24ef-4f75-baca-704e77e4bcb9)

Please help us with disabling this Vercel website as soon as possible.

Kind regards.

### Example

[REDACTED]

### Steps to Reproduce

Hi team! The following URL is being used to perform a phishing attack:

[REDACTED]

It is a multi wallet phishing kit stealing seed phrases from people's crypto wallets:
![image](https://github.com/vercel/community/assets/96588126/58ffc0ca-24ef-4f75-baca-704e77e4bcb9)

Please help us with disabling this Vercel website as soon as possible.

Kind regards.

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this! If you haven't already, please send a copy of this report to [abuse@vercel.com](mailto:abuse@vercel.com) so it can be properly tracked in our system. I'll let the team know to be on the lookout for your email

**Comment by javierpignata** [comment]
Hi! Already did, but you are not answering emails since many weeks :(
Thanks for replying!

Kind regards.

**Reply by amyegan** [reply]
Thanks for your patience. The team is working to address a high volume of reports as quickly as possible. They have let me know that they have taken action and you should see a new notification about your report.

**Reply by javierpignata** [reply]
Thank you! So much appreciated <3

Have a nice day!
